seagence/README.md

```diff
@@ -0,0 +1,56 @@
+
+Seagence integrates with Datadog using an API-based integration. Seagence's tiny Java agent is attached to your production application and posts collected data (ExecutionPaths, errors/exceptions, and contextual information) to the Seagence backend. Continuously analyzing the data stream from Seagence's Java agent, Seagence's backend, using ExecutionPath technology, detects defects when they occur, including their root cause, in real-time. This eliminates the need for debugging and troubleshooting. Using a Rest API, Seagence then posts detected defects and their root cause as Datadog Events to the Datadog backend. You can find the details of the defects in the "Seagence - Defects Overview" dashboard or you can set up a Datadog Monitor to trigger alerts. More details about defects can be found on [SeagenceWeb][2]. With a Seagence provided defect and the root cause in hand, you can fix your broken code.
+
+### Usage
+
+The Seagence integration comes with a dashboard called "Seagence - Defects Overview". The top widget shows the defects timeline and the bottom widget shows a list of defects including their root cause exception with complete stack trace. In the defects timeline view, defects are shown as vertical red bars. You can click any vertical bar to open the context menu. Click "View related events" to open the defects as Datadog Events in the "Events Explorer" dashboard.
+
+## Setup
+
+### Installation
+
+Visit [Seagence][1] to sign up for free. Once registered, you can initiate the connection between Seagence and Datadog in 2 ways:
+1. Find the Seagence tile in Datadog's Integrations view and click **Install Integration**. Click **Connect** on the tile, which guides you through the Datadog OAuth2 flow to grant Seagence access to post Events to your Datadog account.
+2. Alternatively, log in into your Seagence account and go to **Settings** > **Integrations**. Add a Datadog connection and follow the prompts to set up OAuth2 and grant Seagence access to post Events to your Datadog account.
+
+### Configuration
+
+Using the `-javaagent` option, attach Seagence's Java agent to your application. Download the Java agent from your Seagence account. For more information, visit [getting started][3] on [Seagence][1].
+
+### Uninstallation
+
+To remove the Datadog integration from Seagence: 
+1. Navigate to the Seagence's Integrations page and click **Revoke**.
+1. Uninstall the integration from Datadog by clicking **Uninstall Integration**. Once you uninstall the integration, any previous authorizations are revoked.
+1. Ensure that all API keys associated with the integration have been disabled by searching for the integration name on the API Keys management page.
+1. Remove the `-javaagent` option from your application's Java runtime parameters. For more information, visit [getting started][3] on [Seagence][1].
```

Comment:
The linked doc does not provide uninstallation instructions - can you update please?

Reply:
The only uninstallation step is to remove -javaagent option. So I removed the 2nd sentence.